src/GovUk.Education.ExploreEducationStatistics.Content.Model/Import.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using GovUk.Education.ExploreEducationStatistics.Common.Converters;
+using Newtonsoft.Json;
+using static GovUk.Education.ExploreEducationStatistics.Content.Model.ImportStatus;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model
+{
+    public class Import
+    {
+        private static readonly Dictionary<ImportStatus, double> ProcessingRatios = new Dictionary<ImportStatus, double>
+        {
+            {STAGE_1, .1},
+            {STAGE_2, .1},
+            {STAGE_3, .1},
+            {STAGE_4, .7},
+            {CANCELLING, 1},
+            {CANCELLED, 1},
+            {COMPLETE, 1},
+        };
+
+        public Guid Id { get; set; }
```

Comment:
I originally tried to do this with no `Id` field and have `[Key]` on `FileId` but it failed and I was struggling to see why. I now realise this is because at the time I didn't have the explicit config in `ContentDbContext` for the one-to-one relationship so it was being treated as one-to-many (I realised when I saw the original foreign key indexes being generated weren't unique).

It does seem like this is now possible by just using `[Key]` on `FileId` but this is going to be a big change and I'm not sure it's worth it, plus not sure about the `DataImportErrors`, i.e. should they be related to a `DataImport` or to `File`.